css-fonts/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Macondofamily=Roboto:wght@100display=swap" rel="stylesheet">
```

Comment:
Your Google font URL is mangled and isn't loading the fonts correctly:

---

![image](https://user-images.githubusercontent.com/16143771/167954299-f845fced-c755-4475-9231-e2f895bdd5b3.png)

Reply:
Not too sure how to fix it. I broke it into two different links to try and not mangle it.

Reply:
@KrisMackenroth go back and grab the URL from the Google Fonts site again.